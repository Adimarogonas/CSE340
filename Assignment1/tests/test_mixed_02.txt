
9999x00000000001x08

