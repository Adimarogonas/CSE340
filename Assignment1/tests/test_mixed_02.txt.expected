{9999 , NUM , 2}
{x00000000001x08 , ID , 2}
{ , END_OF_FILE , 5}
